packages/graphiql-toolkit/src/storage/history.ts

```diff
@@ -28,6 +28,7 @@ export class HistoryStore {
   private shouldSaveQuery(
     query?: string,
     variables?: string,
+    // extensions?: string, // dz todo: ask about saving strategy for extensions
```

Comment:
In this method, there is logic to determine whether a query should be saved depending on the difference to the last saved query.

I would prefer to not add more logic for extensions here, but I wanted to highlight this in case you had something in mind.

Reply:
ah so - this will be important for extensions because it's how unique tabs are determined, and how extension input values can be saved in history and/or active tabs. you could leave this out for now, but then the lack of persistence will be reported as a bug knowing the expectations of users